webpack.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
         loader: `ts-loader`,
         options: {
           compilerOptions: {
-            module: `es2020`,
+            module: `ES2020`,
```

Comment:
I wonder if this should read from tsconfig.json here to keep them in sync?

Reply:
That's a good idea however my next PR will replace webpack with esbuild so they shouldn't have time to go out of sync.